packages/docs/docs/contracts-api/unirep-voting-sol.md

```diff
@@ -0,0 +1,102 @@
+---
+title: UniRep Voting contract
+---
+
+This is an example contract that utilizes the core UniRep contract and the Voting Prize NFT contract. `UnirepoVoting.sol` is initialized with the number of projects and `VotingPrizeNFT.sol`. 
```

Comment:
You could add:
```
It can help users anonymously join a hackathon project, anonymously vote, and anonymously claim prize.
```
to distinguish this Voting.sol from others voting protocols